csm_web/frontend/src/components/Policies.tsx

```diff
@@ -0,0 +1,498 @@
+import React from "react";
+
+export default class Policies extends React.Component {
+  render(): React.ReactNode {
+    // see SECTIONS doc below to understand what is going on here
+    const intro = SECTIONS.introduction;
+    const htsu = SECTIONS.howToSignUp;
+    const guides = SECTIONS.guidelines;
+    const comms = SECTIONS.communications;
+    const aff = SECTIONS.affinitySections;
+    const forUnit = SECTIONS.csmForUnit;
+    const faq = SECTIONS.faq;
+    return (
+      <div>
+        <div className="sections">
+          <div>
+            <h1>{intro.title}</h1>
+            <p>
+              <a href={intro.body.p1_a.link}>{intro.body.p1_a.text}</a>
+              {intro.body.p1}
+            </p>
+            <p>{intro.body.p2}</p>
+          </div>
+
+          <div className="divider"></div>
+
+          <div>
+            <h1>{htsu.title}</h1>
+            <p>
+              {htsu.body.p1_1}
+              <a href={htsu.body.p1_a.link}>{htsu.body.p1_a.text}</a>
+              {htsu.body.p1_2}
+              <b>{htsu.body.p1_b}</b>
+              {htsu.body.p1_3}
+            </p>
+            <p>{htsu.body.p2}</p>
+            <p>{htsu.body.p3}</p>
+          </div>
+
+          <div className="divider"></div>
+
+          <div>
+            <h1> {guides.title} </h1>
+            <ol>
+              <li>
+                {guides.body.i1_1}
+                <b> {guides.body.i1_b} </b>
+              </li>
+              <li>{guides.body.i2}</li>
+              <li>{guides.body.i3}</li>
+              <li>
+                {guides.body.i4_1}
+                <b> {guides.body.i4_b} </b>
+                {guides.body.i4_2}
+              </li>
+              <li>{guides.body.i5}</li>
+            </ol>
+          </div>
+
+          <div className="divider"></div>
+
+          <div>
+            <h1> {comms.title} </h1>
+            <p>
+              {comms.body.p1_1}
+              <a href={comms.body.p1_a.link}>{comms.body.p1_a.text}</a>
+              {comms.body.p1_2}
+            </p>
+            <p>
+              {comms.body.p2_1}
+              <a href={comms.body.p2_a.link}>{comms.body.p2_a.text}</a>
+              {comms.body.p2_2}
+            </p>
+          </div>
+
+          <div className="divider"></div>
+
+          <div>
+            <h1> {aff.title}</h1>
+            <p> {aff.body.p1} </p>
+            <ol>
+              <li>
+                <b> {aff.body.i1_b} </b>
+                {aff.body.i1_2}
+                <p> {aff.body.i1_i1} </p>
+                <p> {aff.body.i1_i2} </p>
+                <p> {aff.body.i1_i3} </p>
+                <p> {aff.body.i1_i4} </p>
+              </li>
+              <li>
+                <b> {aff.body.i2_b} </b>
+                {aff.body.i2_2}
+              </li>
+              <li>
+                <b> {aff.body.i3_b} </b>
+                {aff.body.i3_2}
+              </li>
+            </ol>
```

Comment:
I'd prefer these lists to be unordered lists rather than ordered lists, to fit in more with the rest of the document.

Reply:
Resolved, converted all lists here to ul's